bedrock/exp/templates/exp/firefox/welcome/base.html

```diff
@@ -19,7 +19,7 @@
 <main class="mzp-t-firefox">
   <header class="c-page-header">
     <div class="mzp-l-content c-page-header-inner">
-      {{ high_res_img('logos/firefox/logo-quantum-full-wordmark.png', {'alt': 'Firefox', 'width': '231', 'height': '40', 'class': 'c-header-logo'}) }}
+      {{ high_res_img('logos/firefox/quantum/logo-word-hor-lg.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-header-logo'}) }}
```

Comment:
Thanks for updating this to use the firefox/quantum logo file 🔥 

I think you already know this needs to include the `img/` now.